xmake/plugins/lua/xmake.lua

```diff
@@ -56,8 +61,33 @@ task("lua")
                 import("scripts." .. name).main(unpack(option.get("arguments") or {}))
             end
         else
+            local replhistory
+            if enable_readline then
+                -- clear history
+                readline.clear_history()
+
+                -- load history
+                replhistory = history.load("replhistory") or {}
+                for _, ln in ipairs(replhistory) do
+                    readline.add_history(ln)
+                end
+            end
+
             -- enter interactive mode
             sandbox.interactive()
+
```

Comment:
If we don't look at implementation of `sandbox.interactive()`, we may not know that `readline` and `sandbox.interactive()` have a relationship in lua action sandbox.